src/hb-aat-layout-trak-table.hh

```diff
@@ -64,21 +64,71 @@ struct TrackData
     return_trace (c->check_struct (this));
   }
 
-  protected:
+  public:
```

Comment:
It's preferred that you add useful methods and keep things protected, than public.

Reply:
Sure

Reply:
Done